c++/src/capnp/compiler/generics.c++

```diff
@@ -584,6 +600,11 @@ kj::Maybe<BrandedDecl> BrandScope::compileDeclExpression(
     case Expression::IMPORT: {
       auto filename = source.getImport();
       KJ_IF_MAYBE(decl, resolver.resolveImport(filename.getValue())) {
+        errorReporter.reportResolution(Resolution {
```

Comment:
Note: 
For example, in the following schema:
```
using Common = import "common.capnp";
```
The resolution is:
```
import "common.capnp"
```

Reply:
You just mean, the `import` keyword is considered part of the identifier, so jump-to-declaration will work even when the cursor is on `import` and not actually on `"common.capnp"`? I think that's fine.